plugins/UM3NetworkPrinting/src/Network/ZeroConfClient.py

```diff
@@ -0,0 +1,135 @@
+# Copyright (c) 2018 Ultimaker B.V.
```

Comment:
This file is nice. Should have done this a long time ago.

Gotta update that copyright though. 😛